client/src/templates/Challenges/redux/index.js

```diff
@@ -158,6 +158,11 @@ export const challengeFilesSelector = state => state[ns].challengeFiles;
 export const challengeMetaSelector = state => state[ns].challengeMeta;
 export const challengeTestsSelector = state => state[ns].challengeTests;
 export const consoleOutputSelector = state => state[ns].consoleOut;
+export const completedChallengesIds = state => {
```

Comment:
```suggestion
export const completedChallengesIds = state =>
  completedChallengesSelector(state).map(node => node.id);
```
Might as well do this in one go.